docs/react/guides/installing-tailwindcss-in-web-react-nextjs-apps.md

```diff
@@ -0,0 +1,71 @@
+# Installing TailwindCss in Web/React/Next.js Apps
+
+This guide will serve as a quickstart to installing TailwindCss in your Web/React/Next.js app.
+
+For more in-depth look on this topic, be sure to check out our blog post on [Setting up Next.js to use Tailwind with Nx](https://blog.nrwl.io/setup-next-js-to-use-tailwind-with-nx-849b7e21d8d0).
+
+## Step 1: Install Tailwind Dependencies
+
+`npm install tailwindcss@latest postcss@latest autoprefixer@latest`
+
+This will install the requisite tailwind dependencies.
+
+## Step 2: Intialize Tailwind
+
+In this step, we'll create a `postcss.config.js` and a `tailwind.config.js` file specific to the application we'd like to introduce tailwind to.
+
+The simplest way to do this uses the Tailwind CLI, and can be done via:
```

Comment:
Avoid usage of words like `via`, `etc`, `e.g.` as they aren't common across languages
```suggestion
The simplest way to do this uses the Tailwind CLI, and can be done with:
```